### PR TITLE
algocfg: Fix algocfg get for non-string parameters.

### PR DESCRIPTION
**Pull request by winder**
## Summary

algocfg was returning values like `%!s(uint64=16)` and `%!s(bool=true)`, with this change those are fixed to return `16` and `true`.

## Test Plan

Seemed silly to unit test a built in function, but  I went ahead and did it anyway.

### CODE REVIEW

**Review by cce — DISMISSED** [review]
LGTM, there is a partitiontest linter warning though

File: cmd/algocfg/getCommand_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPrint(t *testing.T) {
+	testcases := []struct {
+		input    interface{}
```
Comment (brianolson) [inline cmd/algocfg/getCommand_test.go:34]: Make this "Input" ...

File: cmd/algocfg/getCommand_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			expected: "true",
+		},
+		{
+			input:    time.Second,
+			expected: "1s",
+		},
+	}
+	for i, tc := range testcases {
+		t.Run(fmt.Sprintf("test %d", i), func(t *testing.T) {
+			var buf bytes.Buffer
```
Comment (brianolson) [inline cmd/algocfg/getCommand_test.go:56]: and around here `val, err := getObjectProperty(tc, "Input")`
Then Fprintf or Sprintf("%v", val)

**Review by brianolson — COMMENTED** [review]
I think the unit test would be more interesting if it incorporated `getObjectProperty`. Maybe getObjectProperty changes to encompass Sprintf("%v",...) and return the property stringified, then that function is fully unit testable.

File: cmd/algocfg/getCommand_test.go (hunk 2 above)
Comment (winder) [reply cmd/algocfg/getCommand_test.go:56]: Using the test case as an input to the test function is genius.

File: cmd/algocfg/getCommand.go [hunk 3]
```diff
@@ -51,29 +51,29 @@ var getCmd = &cobra.Command{
...
-			val, err := getObjectProperty(cfg, getParameterArg)
+			val, err := serializeObjectProperty(cfg, getParameterArg)
 			if err != nil {
 				reportWarnf("Error retrieving property '%s' - %s", getParameterArg, err)
 				anyError = true
 				return
 			}
 
-			fmt.Printf("%s", val)
+			fmt.Printf(val)
```
Comment (brianolson) [inline cmd/algocfg/getCommand.go:61]: should be just Print() for a plain string

**Review by brianolson — COMMENTED** [review]
One last tweak then good to go

File: cmd/algocfg/getCommand.go (hunk 3 above)
Comment (winder) [inline cmd/algocfg/getCommand.go:61]:
```suggestion
			fmt.Print(val)
```

**Review by brianolson — APPROVED** [review]

**Pull request merged**